src/mono/wasm/runtime-test.js

```diff
@@ -388,4 +388,7 @@ var App = {
 			fail_exec ("Unhanded argument: " + args [0]);
 		}
 	},
-};
+	call_test_method: function (method_name, signature, args) {
```

Comment:
Should `signature` parameter be used anywhere?

Reply:
Yes, I have a change for that locally.  Thanks